src/web-integration-step-by-step.md

```diff
@@ -0,0 +1,108 @@
+* [Single Sign On](/nhsapp-developer-documentation/web-integration-step-by-step/#sso)
+* [Site Links](/nhsapp-developer-documentation/web-integration-step-by-step/#sitelinks)
+* [Limitations](/nhsapp-developer-documentation/web-integration-step-by-step/#limitations)
+
+## Hiding Headers and Footers <a name="headersandfooters"></a>
+
+In order to fit in with the styling of the NHS App - suppliers are required to hide headers and footer - which will be superseded by the NHS App native headers and footers.
+
+The NHS App will identify itself by adding a custom string in the format `nhsapp-android/<nhsapp-version>` or `nhsapp-ios/<nhsapp-version>` to the end of the user agent header when making requests (where nhsapp-version is the current version of the native NHS App, e.g. 1.29.0).
+
+### Examples
+
+| Device  | App Version | Custom User Agent                                                                                                                               |
+|---------|-------------|-------------------------------------------------------------------------------------------------------------------------------------------------|
+| Pixel 2 | 1.26.2      | Mozilla/5.0 (Linux; Android 10; Pixel 2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.116 Mobile Safari/537.36 nhsapp-android/1.26.2 |
+| Phone   | 1.25.0      | Mozilla/5.0 (iPhone; CPU iPhone OS 6_1_2 like Mac OS X) AppleWebKit/536.26 (KHTML, like Gecko) Mobile/10B146 nhsapp-ios/1.25.0                  |
+
+One option to detect the device type is to use the  [NHS App JS API isOpenInNHSApp function](/nhsapp-developer-documentation/js-api-specification/#isOpenInNHSApp)
+
+## NHS Login Single Sign On <a name="sso"></a>
+
+Logged in users of the application will be passed to third party services using [NHS Login Single Sign On flow](https://nhsconnect.github.io/nhslogin/single-sign-on/). 
+
+Single Sign on will be achieved by passing the NHS App JWT token via a query parameter. The NHS App will allow a supplier to provide one or more url paths to be integrated with.
+
+Suppliers will provide the following details to the NHS App prior to an integration:
+
+* Base Url (for each environment stage)
+* Journey Path(s) 
+
+The NHS App will then redirect to the supplier using the following url format 
+
+`<baseurl>/<journey-path>?assertedLoginIdentity=<sso-jwt>`
+
+
+E.g. `https://supplier1.com/hospital-appointments?assertedLoginIdentity=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c`
+
+The `assertedLoginIdentity` query parameter will be added as the last parameter, so it is possible for the journey path to include query parameters.
+
+### NHS Login - Authorize call
+The NHS requires suppliers to pass prompt=none to the NHS Login `/authorize` endpoint. This will ensure that user is shown an error page if there is an SSO issue.
```

Comment:
```suggestion
The NHS requires suppliers to pass `prompt=none` to the NHS Login `/authorize` endpoint. This will ensure that user is shown an error page if there is an SSO issue.
```